### PR TITLE
(DOCSP-16839) Backfills docs content for organizations apiKeys and projects apiKeys commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for organizations apiKeys and projects apiKeys commands.

**Note: I assumed that the value values for Atlas CLI are the same as they are for MongoCLI version of these commands. If that's not accurate, please let me know and I will point them to use different flag descriptions.**

_Jira ticket:_

https://jira.mongodb.org/browse/DOCSP-16839
https://jira.mongodb.org/browse/DOCSP-16840
https://jira.mongodb.org/browse/DOCSP-16841
https://jira.mongodb.org/browse/DOCSP-16843
https://jira.mongodb.org/browse/DOCSP-16844
https://jira.mongodb.org/browse/DOCSP-16845
https://jira.mongodb.org/browse/DOCSP-16846
https://jira.mongodb.org/browse/DOCSP-16847
https://jira.mongodb.org/browse/DOCSP-16856
https://jira.mongodb.org/browse/DOCSP-16857
https://jira.mongodb.org/browse/DOCSP-16858
https://jira.mongodb.org/browse/DOCSP-16859

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/iam/organizations/apikeys/accesslists/delete.go [hunk 1]
```diff
@@ -54,11 +55,13 @@ func DeleteBuilder() *cobra.Command {
...
 		Use:     "delete <ID>",
 		Aliases: []string{"rm"},
-		Short:   "Delete an IP access list from your API Key.",
+		Short:   "Remove the specified IP access list entry from your API Key.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"IDDesc": "Access list identifier.",
+			"IDDesc": "IP or CIDR address that you want to remove from the access list. If the entry includes a subnet mask, such as 192.0.2.0/24, use the URL-encoded value %2F for the forward slash /.",
 		},
+		Example: fmt.Sprintf(`  # Remove the IP address 192.0.2.0 from the access list for the API key with the ID 5f24084d8dbffa3ad3f21234 in the organization with the ID 5a1b39eec902201990f12345:
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/accesslists/delete.go:63]: Should we also include an example that shows usage of the URL-encoded value for the forward slash?

File: internal/cli/iam/organizations/apikeys/delete.go [hunk 2]
```diff
@@ -53,11 +54,14 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <ID>",
 		Aliases: []string{"rm"},
-		Short:   "Delete an Organization API Key.",
+		Short:   "Remove the specified API key for your organization.",
+		Long:    fmt.Sprintf(`To view possible values for the ID argument, run %s organizations apiKeys list.`, cli.ExampleAtlasEntryPoint()),
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"IDDesc": "API key identifier.",
+			"IDDesc": "Unique 24-digit string that identifies your API key.",
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/delete.go:61]:
```suggestion
			"IDDesc": "Unique 24-hexadecimal digit string that identifies the organization's API key.",
```

File: internal/cli/iam/organizations/apikeys/delete.go [hunk 3]
```diff
@@ -53,11 +54,14 @@ func DeleteBuilder() *cobra.Command {
...
 		},
+		Example: fmt.Sprintf(`  # Remove the organization API key with the ID 5f24084d8dbffa3ad3f21234 in the organization with the ID 5a1b39eec902201990f12345:
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/delete.go:63]:
```suggestion
		Example: fmt.Sprintf(`  # Remove the organization API key with the ID 5f24084d8dbffa3ad3f21234 for the organization with the ID 5a1b39eec902201990f12345:
```

File: internal/cli/iam/organizations/apikeys/describe.go [hunk 4]
```diff
@@ -61,10 +62,13 @@ func DescribeBuilder() *cobra.Command {
 		Use:     "describe <ID>",
 		Aliases: []string{"show"},
 		Args:    require.ExactArgs(1),
-		Short:   "Get a specific Organization API Key.",
+		Short:   "Return the details for the specified API key for your organization.",
+		Long:    fmt.Sprintf(`To view possible values for the ID argument, run %s organizations apiKeys list.`, cli.ExampleAtlasEntryPoint()),
 		Annotations: map[string]string{
-			"IDDesc": "API key identifier.",
+			"IDDesc": "Unique 24-digit string that identifies your API key.",
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/describe.go:68]:
```suggestion
			"IDDesc": "Unique 24-hexadecimal digit string that identifies your API key.",
```

File: internal/cli/iam/organizations/apikeys/describe.go [hunk 5]
```diff
@@ -61,10 +62,13 @@ func DescribeBuilder() *cobra.Command {
...
 		},
+		Example: fmt.Sprintf(`  # Return the JSON-formatted details for the organization API key with the ID 5f24084d8dbffa3ad3f21234 in the organization with the ID 5a1b39eec902201990f12345:
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/describe.go:70]:
```suggestion
		Example: fmt.Sprintf(`  # Return the JSON-formatted details for the organization API key with the ID 5f24084d8dbffa3ad3f21234 for the organization with the ID 5a1b39eec902201990f12345:
```

File: internal/cli/iam/organizations/apikeys/list.go [hunk 6]
```diff
@@ -61,8 +62,10 @@ func ListBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "list",
 		Aliases: []string{"ls"},
-		Short:   "List API Keys for your organization.",
+		Short:   "Return all API keys for your organization.",
 		Args:    require.NoArgs,
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of organization API keys in the organization with the ID 5a1b39eec902201990f12345:
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/apikeys/list.go:67]:
```suggestion
		Example: fmt.Sprintf(`  # Return a JSON-formatted list of organization API keys for the organization with the ID 5a1b39eec902201990f12345:
```

File: internal/usage/usage.go [hunk 7]
```diff
@@ -143,10 +143,10 @@ dbName and collection are required only for built-in roles.`
...
 	AlertConfigAPIKey                         = "Datadog API Key, Opsgenie API Key, or VictorOps API key. Required if the notificationType is DATADOG, OPS_GENIE, or VICTOR_OPS, respectively." //nolint:gosec // This is just a message not a credential
-	APIKey                                    = "API Key."
-	DatadogAPIKey                             = "Datadog API key that allows Atlas to access your Datadog account."               //nolint:gosec // This is just a message not a credential
-	OpsgenieAPIKey                            = "Opsgenie API key that allows Atlas to access your Opsgenie account."             //nolint:gosec // This is just a message not a credential
-	VictorOpsAPIKey                           = "Splunk On-Call API key that allows Atlas to access your Splunk On-Call account." //nolint:gosec // This is just a message not a credential
+	APIKey                                    = "Unique 24-digit string that identifies your API key."                                                                                          //nolint:gosec // This is just a message not a credential
```
Comment (kanchana-mongodb) [inline internal/usage/usage.go:146]:
```suggestion
	APIKey                                    = "Unique 24-hexadecimal digit string that identifies your API key."                                                                                          //nolint:gosec // This is just a message not a credential
```

File: internal/usage/usage.go [hunk 8]
```diff
@@ -246,9 +246,9 @@ dbName and collection are required only for built-in roles.`
 	PartitionFields                           = "Fields to use to partition data. You can specify up to two frequently queried fields, separated by a comma, in the following format: <fieldname>:<datatype>. The data type must match the data type of the field in the document."
 	ArchiveAfter                              = "Number of days after which to archive cluster data."
 	TargetProjectID                           = "Unique identifier of the project that contains the destination cluster for the restore job. You must specify a targetProjectId for automated restores."
-	APIAccessListIPEntry                      = "IP address to be allowed for a given API key."
+	APIAccessListIPEntry                      = "IP address that you want to add to the access list for your API key. To add more than one IP address, you can specify each address with a separate ip flag or specify the addresses in a comma-separated list with one ip flag. You can't set both cidr and ip in the same command."
```
Comment (kanchana-mongodb) [inline internal/usage/usage.go:249]:
```suggestion
	APIAccessListIPEntry                      = "IP address that you want to add to the access list for your API key. To add more than one IP address, you can specify each address with a separate ip flag or specify all the addresses as a comma-separated list using one ip flag. You can't set both ip and cidr in the same command."
```

File: internal/usage/usage.go [hunk 9]
```diff
@@ -246,9 +246,9 @@ dbName and collection are required only for built-in roles.`
...
 	NetworkAccessListIPEntry                  = "IP address to be allowed to access the deployment."
-	AccessListCIDREntry                       = "Access list entry in CIDR notation to be added for a given API key."
+	AccessListCIDREntry                       = "Access list entry in CIDR notation to be added for your API key. To add more than one entry, you can specify each entry with a separate cidr flag or specify the entries in a comma-separated list with one cidr flag. You can't set both cidr and ip in the same command."
```
Comment (kanchana-mongodb) [inline internal/usage/usage.go:251]:
```suggestion
	AccessListCIDREntry                       = "Access list entry in CIDR notation to be added for your API key. To add more than one entry, you can specify each entry with a separate cidr flag or specify all the entries as a comma-separated list using one cidr flag. You can't set both cidr and ip in the same command."
```

File: internal/cli/iam/organizations/apikeys/accesslists/delete.go (hunk 1 above)
Comment (sarahsimpers) [reply internal/cli/iam/organizations/apikeys/accesslists/delete.go:63]: Unfortunately, I tried to do that and it caused an error as it interpreted the % as part of the code. But if engineering knows how I might include it without causing issues I can do that

**Review by sarahsimpers — COMMENTED** [review]
Hi @kanchana-mongodb, thanks for reviewing! I made all of your suggested changes except for the addition of "hexadecimal"... I'm trying to keep descriptions as short as possible since they are shown in the CLI itself, so I have removed some of the words we used in OpenAPI definitions where I felt they didn't add a lot of value in identifying flag values. Back to you!

File: internal/usage/usage.go (hunk 7 above)
Comment (andreaangiolillo) [reply internal/usage/usage.go:146]: [q] Why did you add this usage? is this for public or private keys? Where is it used?

Comment (sarahsimpers) [reply internal/usage/usage.go:146]: Hi @andreaangiolillo, these 4 items already existed in the usage file, I just changed the descriptions for all of them and the make fmt added some space between the description and the nolint note, so they're showing as removed and re-added. This particular example just replaces `API Key.` with `Unique 24-digit string that identifies your API key.` for the existing item.

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**